docs/UserGuide.adoc

```diff
@@ -47,14 +51,31 @@ Interested? Jump to the <<Quick Start>> to get started.
 // comment: add a preface to everything, need to orientate user
 
 
-== Quick Start
-(ADD PREFACE)
-(PROVIDE LINKS TO RELEVANT DOWNLOADABLES)
+== How to use User Guide
```

Comment:
Ms anitah might not like how 'this' has been omitted. "How to use this guide" or "How to use this user guide". No need to capitalize User and Guide because they are not proper nouns